src/test/java/org/folio/roles/it/PermissionsUserIT.java

```diff
@@ -77,34 +83,38 @@ private static Stream<Arguments> getPermissionsUserDataProvider() {
 
     return Stream.of(
       arguments(userId1, true, List.of(
-        "module.foo.item.post", "plugin.foo.item.get", "ui-foo.item.delete", "ui-foo.item.put")),
+          "module.foo.item.post", "plugin.foo.item.get", "ui-foo.item.delete", "ui-foo.item.put"),
+        List.of("ui-foo.item.*", "module.foo.item.post")),
       arguments(userId1, false, List.of(
         "foo.item.delete", "foo.item.get", "foo.item.post", "foo.item.put",
-        "module.foo.item.post", "plugin.foo.item.get", "ui-foo.item.delete", "ui-foo.item.put")),
+        "module.foo.item.post", "plugin.foo.item.get", "ui-foo.item.delete", "ui-foo.item.put"), List.of()),
```

Comment:
```suggestion
        "module.foo.item.post", "plugin.foo.item.get", "ui-foo.item.delete", "ui-foo.item.put"), emptyList()),
```

anywhere where using List.of()